### PR TITLE
Low Latency DASH web documentation

### PR DESCRIPTION
**Pull request by CaitlinOCallaghan**
Documentation for low latency DASH support

Sphinx previews:

image
image

### CODE REVIEW

File: docs/source/tutorials/low_latency.rst [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    export UPLOAD_URL=http://localhost:6767/ll-dash
+
+    # Go
+    packager \
+        "input=${PIPE},stream=audio,init_segment=${UPLOAD_URL}_init.m4s,segment_template=${UPLOAD_URL}/bigbuckbunny-audio-aac-\$Number%04d\$.m4s" \
+        "input=${PIPE},stream=video,init_segment=${UPLOAD_URL}_init.m4s,segment_template=${UPLOAD_URL}/bigbuckbunny-video-h264-450-\$Number%04d\$.m4s" \
+        --io_block_size 65536 \
+        --segment_duration 2 \
+        --is_low_latency_dash=true \
+        --utc_timings "urn:mpeg:dash:utc:http-xsdate:2014"="https://time.akamai.com/?iso" \
```
Comment (CaitlinOCallaghan) [inline docs/source/tutorials/low_latency.rst:69]: Is there a UTCTiming value that we would prefer to use other than the Akamai one?

File: docs/source/tutorials/low_latency.rst [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+For low latency to be achieved, the processes handling Shaka Packager's output, such as the server and player,
+must support LL-DASH streaming.
+
+Delivery Pipeline
+=================
+Shaka Packager will upload the LL-DASH content to the specified output via HTTP chunked transfer encoding.
+The server must have the ability to handle this type of request. If using a proxy or shim for cloud authentication,
+these services must also support HTTP chunked transfer encoding.
+
+Examples of supporting content delivery systems:
```
Comment (CaitlinOCallaghan) [inline docs/source/tutorials/low_latency.rst:86]: Does GCP have any tools that support Chunked Transfer Encoding? Unfortunately, I am only familiar with AWS.

File: docs/source/options/dash_options.rst [hunk 3]
```diff
@@ -95,3 +95,8 @@ DASH options
 
     If enabled, allow adaptive switching between different codecs, if they have 
     the same language, media type (audio, video etc) and container type.
+
+--is_low_latency_dash
```
Comment (joeyparrish) [inline docs/source/options/dash_options.rst]: Same comment I had on the streamer side: let's not use "is".  :-)

File: docs/source/tutorials/low_latency.rst [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+************
+Introduction
+************
+
+If ``--is_low_latency_dash`` is enabled, low latency DASH (LL-DASH) packaging will be used.
+
+This will reduce overall latency by ensuring that the media segments are chunk encoded and delivered via an aggregating response.
+The combination of these features will ensure that overall latency can be decoupled from the segment duration.
+For low latency to be achieved, the output of Shaka Packager must be combined with a delivery system which can chain together a set of aggregating responses, such as chunked transfer encoding under HTTP/1.1 or a HTTP/2 or HTTP/3 connection.
+The output of Shaka Packager must be played DASH client which understands the availabilityTimeOffset MPD value.
```
Comment (joeyparrish) [inline docs/source/tutorials/low_latency.rst]: missing words: "... must be played **in** (or **with**, your call) **a** DASH client ..."

File: docs/source/tutorials/low_latency.rst (hunk 1 above)
Comment (joeyparrish) [reply docs/source/tutorials/low_latency.rst:69]: LGTM

File: docs/source/tutorials/low_latency.rst (hunk 2 above)
Comment (joeyparrish) [reply docs/source/tutorials/low_latency.rst:86]: Sadly, not yet.

**Review by joeyparrish — CHANGES_REQUESTED** [review]
Looks great otherwise.  Thanks!

**Comment by ngarg17** [comment]
Besides Joey's suggested changes it looks good 👍

File: docs/source/options/dash_options.rst (hunk 3 above)
Comment (CaitlinOCallaghan) [reply docs/source/options/dash_options.rst]: Good call! I just updated the naming here. I also made a PR to address the name change in the rest of the files: https://github.com/CaitlinOCallaghan/shaka-packager/pull/7

**Review by stephenlindauer — APPROVED** [review]
Changes look good!

**Pull request merged**